ts-packages/web/src/app/(social)/_components/home-news.tsx

```
@@ -0,0 +1,50 @@
+        {newsData.map((item) => (
+          <Col
+            onClick={() => handleNewsNavigation(item.id)}
+            key={`news-${item.id}`}
+            className="py-2.5 cursor-pointer"
+          >
+            <h4 className="text-base/[25px] tracking-[0.5px] align-middle font-medium text-foreground">
+              {item.title}
+            </h4>
+            <div
+              className="text-sm/[20px] align-middle font-light line-clamp-2 whitespace-normal text-card-meta"
+              dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)